dotnet/samples/KernelSyntaxExamples/KernelSyntaxExamples.csproj

```diff
@@ -83,13 +75,25 @@
     <PackageReference Include="Microsoft.Net.Compilers.Toolset" />
   </ItemGroup>
   <ItemGroup>
-    <EmbeddedResource Include="Resources\*">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </EmbeddedResource>
+    <Compile Remove="RepoUtils\PlanExtensions.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="Resources\Agents\ParrotAgent.yaml" />
+    <None Remove="Resources\Agents\ToolAgent.yaml" />
+    <None Remove="Resources\GenerateStory.yaml" />
+    <None Remove="Resources\GenerateStoryHandlebars.yaml" />
+    <None Remove="Resources\travelinfo.txt" />
   </ItemGroup>
   <ItemGroup>
     <None Include="Plugins\ApiManifestPlugins\**\apimanifest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Resources\*">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Resources\Agents\ParrotAgent.yaml" />
+    <EmbeddedResource Include="Resources\Agents\ToolAgent.yaml" />
```

Comment:
Should we use wildcards or can the other statement we extended to include subdirectories?

Reply:
I wondered on whether to use a greedy expression for sub-directories.  I think I'd prefer `Resources\Agents\*` for control.  

I guess when I saw the project was already asymetrical (with the `None` entries), it chose not to overthink.

Reply:
Just noticed, looks like `**` is greedy:

![image](https://github.com/microsoft/semantic-kernel/assets/66376200/d6e36785-e594-40f6-b519-3d15909bf19e)

Reply:
Ok, I don't think the "None" entries are doing anything...removed.